product_examples/errors-and-omissions/products/errors-omissions/policy/taxes/taxes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "taxes": [
+    {
+      "name": "sales",
+      "displayName": "Sales Tax"
```

Comment:
![Screen Shot 2019-06-25 at 11 59 46 AM](https://user-images.githubusercontent.com/3634095/60125527-cfb5a580-9740-11e9-96ce-08147a20ed43.png)

The invoice shows "A legal liability" and "A sales tax". Can we remove the "A"?